doc/ref_model/chapters/chapter04.md

```diff
@@ -282,43 +282,75 @@ The idea of the Cloud Infrastructure profiles is to have a predefined set of inf
 
```

Comment:
sorry to place this comment here -- it should be at the 4.2 Infrastructure Profiles Catalogue

Here are my suggestions for this Chapter:
4.2 Catalogue  (as it is in the TOC)
Description of the content and organization of this section

4.2.1 Workload Profiles
These Workload Profiles should be defined aligned with RM Chapter 02.

4.2.2 Compute Flavours
keep this about resource sizing cpu, RAM, different types of storage (ephemeral, etc.) and network speeds.

Reply:
@pgoyal01 The proposal is not aligned with TOC listed in #2384 (lets fix this one first here)

<img width="862" alt="image" src="https://user-images.githubusercontent.com/77441414/117291830-61c38480-ae67-11eb-9a5d-29090dabe29e.png">